principal/server.go

```diff
@@ -271,6 +362,22 @@ func (s *Server) Start(ctx context.Context, errch chan error) error {
 	}
 	log().Infof("AppProject informer synced and ready")
 
+	// Start resource proxy if it is enabled
+	if s.resourceProxy != nil {
+		_, err = s.resourceProxy.Start(s.ctx)
+		if err != nil {
+			return fmt.Errorf("unable to start KubeProxy: %w", err)
```

Comment:
We have used the name Kube proxy in some places. I assume it is synonymous with the word resource proxy.

Reply:
Oh, good catch! Yeah, the package was named KubeProxy before I renamed it to ResourceProxy.

Reply:
Renamed all occurrences of KubeProxy to ResourceProxy.